docs/userguides/clis.md

```diff
@@ -158,3 +158,23 @@ def create_account(alias):
     # We know the alias is not yet used in Ape at this point.
     click.echo(alias)
 ```
+
+You can control additional filtering of the alias account options by using one or both of the `account_type` or `account_list` kwargs.
+Use `account_type` to filter the choices by specific types of [AccountAPI](../methoddocs/api.html#ape.api.accounts.AccountAPI).
+Use the `account_list` kwarg to filter the choices by a given sequence of accounts.
+
+```python
+import click
+from ape import accounts
+from ape.cli import existing_alias_argument
+from ape_accounts.accounts import KeyfileAccount
+
+def _valid_accounts():
+   return [a for a in accounts if a.alias.startswith("root_signer_")]
+
+@click.command()
+@existing_alias_argument(account_type=KeyfileAccount, account_list=_valid_accounts())
```

Comment:
also curious if using a key filter function (e.g. `Callable[[AccountAPI], bool]` signature function) would be more flexible

Reply:
That would work! However, it may be a little less efficient. Maybe we can have it accept either a callable or a list?

Reply:
it might be more efficient because it doesn't have to loop `accounts` first

Reply:
Updated! `account_type` can now be a callable, a sequence, or the original Type.

If you already have a list, you avoid looking at accounts entirely:

```python
selected_signer = get_user_selected_account(account_type=safe.local_signers)
```